app/mailers/notification_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class NotificationMailer < ApplicationMailer
+  def release_email
+    @user = params[:user]
+    @grouping = params[:grouping]
+    attachments.inline['markus_logo_big.png'] = File.read(Rails.root.join('app', 'assets', 'images', 'markus_logo_big.png'))
+    mail(to: @user.email, subject: 'MarkUs Notification (' + Rails.configuration.course_name + ') Your marks for ' + @grouping.assignment.short_identifier + ' have released!')
```

Comment:
This is a good start, please internationalize the strings (https://guides.rubyonrails.org/i18n.html). You can add a new subfolder called `config/locales/*/mailers`.

Reply:
Woah, I'll need to look into this more.